<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>player_names</class>
 <widget class="QDialog" name="player_names">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1269</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <widget class="QLabel" name="gametitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Noto Serif</family>
         <pointsize>36</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ENTER YOUR PSEUDOS</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2" rowspan="2">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <layout class="QFormLayout" name="formLayout_4">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <property name="rowWrapPolicy">
            <enum>QFormLayout::WrapLongRows</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="p1">
             <property name="font">
              <font>
               <family>Noto Serif</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Player 1:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="p1pseudo">
             <property name="font">
              <font>
               <family>Noto Serif</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>(Mandatory)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="p2">
             <property name="font">
              <font>
               <family>Noto Serif</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Player 2:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="p2pseudo">
             <property name="font">
              <font>
               <family>Noto Serif</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>(Mandatory)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="p3">
             <property name="font">
              <font>
               <family>Noto Serif</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Player 3:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="p3pseudo">
             <property name="font">
              <font>
               <family>Noto Serif</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>(Mandatory)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="p4">
             <property name="font">
              <font>
               <family>Noto Serif</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Player 4:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="p4pseudo">
             <property name="font">
              <font>
               <family>Noto Serif</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>(Mandatory)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="p5">
             <property name="font">
              <font>
               <family>Noto Serif</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Player 5:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="p5pseudo">
             <property name="font">
              <font>
               <family>Noto Serif</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>(Optional)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="p6pseudo">
             <property name="font">
              <font>
               <family>Noto Serif</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>(Mandatory)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="p7">
             <property name="font">
              <font>
               <family>Noto Serif</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Player 7:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="p7pseudo">
             <property name="font">
              <font>
               <family>Noto Serif</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>(Mandatory)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="p8">
             <property name="font">
              <font>
               <family>Noto Serif</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Player 8:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="p8pseudo">
             <property name="font">
              <font>
               <family>Noto Serif</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>(Optional)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="p6">
             <property name="font">
              <font>
               <family>Noto Serif</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Player 6:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="HSpacer_RightDown">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Warning">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>164</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>164</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>190</red>
               <green>190</green>
               <blue>190</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font">
          <font>
           <family>Noto Serif</family>
           <pointsize>30</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="previous">
         <property name="font">
          <font>
           <family>Noto Serif</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Previous</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="cut_wires_button">
       <property name="font">
        <font>
         <family>Noto Serif</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cut wires</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
